dashboard/config/scripts_json/csp4-2024.script_json

```diff
@@ -10078,5 +10078,45 @@
         "script.name": "csp4-2024"
       }
     }
+  ],
+  "skills": [
+    {
+        key: "variables_declare",
+        description: "Declare variables correctly",
+        evaluation_criteria: "Did the students declare all of the variables in their code correctly?",
+        concept: "Variables"
+      },
+      {
+        key: "variables_name",
+        description: "Name variables according to conventions",
+        evaluation_criteria: "Are there any spaces in variable names? Are there any misspelled variable names? Do variable names follow casing conventions?",
+        concept: "Variables"
+      },
+      {
+        key: "variables_increment",
+        description: "Increment values stored in variables",
+        evaluation_criteria: "Does the student's added code increment the values stored in the variables correctly?",
+        concept: "Variables"
+      },
+  ],
+  "levels_skills": [
```

Comment:
the list of skills associated with a level seem like they should be in the .level file.

the `levels_skills` table associates a skill with a level, not with a script_level. The serialization of a Level object should be stored in a `.level` file (or dsl-defined level file type). the .script_json file owns the serialization of everything from `Unit` down to `ScriptLevel`, but not the `Level` itself.

The way this is written could lead to some unwanted behavior when a level appears in multiple units.